envs/conda-env-sklearn.yml

```diff
@@ -2,19 +2,15 @@ channels:
   - conda-forge
   - nodefaults
 dependencies:
-  - python=3.10
   # additional frameworks
   - xgboost
   - catboost
   - lightgbm
   - faiss-cpu
   - modin-all
-  - intel::scikit-learn-intelex
-  - intel::daal4py
-  - intel::dpctl
```

Comment:
why remove dpctl?

Reply:
Due to current uncertainty of intel conda channel.